### PR TITLE
Windows test runs and documentation

### PR DESCRIPTION
**Pull request by AnoushkaU**
The runs on windows gives memory allocation errors. This has nothing to do with the program options library that was recently added. I have excluded program options completely and checked it, and the error persists. But it runs fine on local windows machines.

### CODE REVIEW

**Review by rafzi — COMMENTED** [review]
Thanks for this! Looking good overall.

I got it working on my machine, but that did require some changes. See comments.

File: README.md [hunk 1]
```diff
@@ -75,6 +77,42 @@ Install the package:
...
+
+	$ mkdir build_dir
+
+Change to the created directory, e.g.,
+
+	$ cd build_dir
+
+Configure the build system:
+
+	$ cmake -DCMAKE_BUILD_TYPE=Release -DETISS_BUILD_MANUAL_DOC=ON -DCMAKE_INSTALL_PREFIX:PATH=X:/install/path ..
```
[the remark this answers is not in the document]
Comment (AnoushkaU) [reply README.md:106]: I did it from powershell and it worked for me automatically. What do you suggest we document here? Should we ask the user to use the developer command prompt as well?

[the remark this answers is not in the document]
Comment (rafzi) [reply README.md:106]: In powershell, can you just call "cl" and it finds the compiler? That does not work for me - only in the vs dev shell. So I would recommend to use that one in the documentation.

For CMake I noticed that I have "CMake tools for Windows" installed. I'm not sure if that is the default, but if not we should recommend to install this. Then we do not have to give instructions how to edit the PATH to point to CMake. This CMake is then also available in the vs dev shell.

https://rafzi.page.link/fYZg

[the remark this answers is not in the document]
Comment (AnoushkaU) [reply README.md:106]: In the developer powershell it finds the compiler if I just type "cl". And cmake I installed using Chocolatey

**Pull request merged**